From af0cd2b5b8dfbc22c2ac85721308ff3e202b754d Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Mon, 9 Jul 2012 15:01:37 +0800
Subject: [PATCH 022/602] ENGR00216063 Config back/front camera by full sensor
 name

Config the ov5640 mini as the back camera since csi cannot
reach 1080p 30 fps requirement

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 arm2_6dq/init.rc     | 2 +-
 arm2_6sl/init.rc     | 2 +-
 imx6/sabresd/init.rc | 4 ++--
 sabreauto_6q/init.rc | 2 +-
 4 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/arm2_6dq/init.rc b/arm2_6dq/init.rc
index e2deb7e..976275e 100755
--- a/arm2_6dq/init.rc
+++ b/arm2_6dq/init.rc
@@ -24,7 +24,7 @@ on boot
     #For landscape mode, orient is 0
     #For portrait mode, orient is 90
     #the android before honycomb are all in portrait mode
-    setprop back_camera_name ov
+    setprop back_camera_name ov5640_mipi
     setprop back_camera_orient 0
 
     # Set OpenGLES version
diff --git a/arm2_6sl/init.rc b/arm2_6sl/init.rc
index ea393ae..0bb5e5e 100755
--- a/arm2_6sl/init.rc
+++ b/arm2_6sl/init.rc
@@ -20,7 +20,7 @@ on boot
     #For landscape mode, orient is 0
     #For portrait mode, orient is 90
     #the android before honycomb are all in portrait mode
-    setprop back_camera_name ov
+    setprop back_camera_name ov5640_mipi
     setprop back_camera_orient 0
 
     # Set OpenGLES version
diff --git a/imx6/sabresd/init.rc b/imx6/sabresd/init.rc
index 3bd0a3b..3ba950d 100755
--- a/imx6/sabresd/init.rc
+++ b/imx6/sabresd/init.rc
@@ -36,9 +36,9 @@ on boot
     #For landscape mode, orient is 0
     #For portrait mode, orient is 90
     #the android before honycomb are all in portrait mode
-    setprop back_camera_name ov
+    setprop back_camera_name ov5640_mipi
     setprop back_camera_orient 0
-    setprop front_camera_name ov
+    setprop front_camera_name ov5642
     setprop front_camera_orient 0
 
     # Set OpenGLES version
diff --git a/sabreauto_6q/init.rc b/sabreauto_6q/init.rc
index 9c7bba6..53a09a6 100644
--- a/sabreauto_6q/init.rc
+++ b/sabreauto_6q/init.rc
@@ -24,7 +24,7 @@ on boot
     #For landscape mode, orient is 0
     #For portrait mode, orient is 90
     #the android before honycomb are all in portrait mode
-    setprop back_camera_name ov
+    setprop back_camera_name ov5640_mipi
     setprop back_camera_orient 0
 
     # Set OpenGLES version
-- 
1.8.0

